src/redturtle/importer/base/transmogrifier/blueprints/logger.py

```diff
@@ -80,13 +80,13 @@ def __iter__(self):
             logger.warning("\nNext objects errored somewhere in the pipeline:")
             for error in self.errored:
                 logger.warning(
-                    '- {path}: {reason}'.format(
-                        path=error['path'], reason=error.get('reason', '')
+                    "- {path}: {reason}".format(
+                        path=error["path"], reason=error.get("reason", "")
                     )
                 )
             # self.log_errors(self.errored)
         # delete validation data from annotations
-        annotations = IAnnotations(self.transmogrifier)
```

Comment:
@cekk sicuro che qui sia la stessa cosa ?

Reply:
serve un posto "volatile" dove appoggiare quelle info.
In alcuni punti con transmogrifier non riesco ad adattare le Annotations e non so bene perché, quindi ho pensato di usare la request come contesto, così finita la request butto tutto